sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_utils/__init__.py

```diff
@@ -43,6 +44,15 @@ def _get_customer_ikey_from_env_var():
     return _CUSTOMER_IKEY_ENV_VAR
 
 
+# TODO: Add environment variable to enable/disable diagnostics
+def _is_diagnostics_enabled():
+    if _is_on_functions():
+        return False
+    if _is_on_app_service() or _is_on_aks():
+        return _is_attach_enabled()
```

Comment:
Just to confirm, this is the same behavior because we only had diagnostic/status logging for attach scenarios before right?